app/components/pan-zoom.jsx

```diff
@@ -261,14 +261,28 @@ class PanZoom extends Component {
   }
 
   rotateClockwise() {
-    const newRotation = this.state.rotation + 90;
+    const newRotation = (this.state.rotation + 90) % 360;
     this.setState({
       rotation: newRotation,
       transform: `rotate(${newRotation} ${this.props.frameDimensions.width / 2} ${this.props.frameDimensions.height / 2})`
     });
   }
 
+  /*
+  TODO: hide this behind an experimental flag
+   */
```

Comment:
```suggestion
```

Non-blocking suggestion, but I think this comment can be deleted since pan-zoom's render method looks for `subjectViewer-freeRotation` in `experimental_tools` right?

Reply:
Ah, good catch! 👍